### DISCUSSION TITLE
Does Magick.Net support Fourier Transforms?

### DISCUSSION THREAD

**Discussion by alex-vanattaresearch** [dlemstra/Magick.NET#1770] [Help]
Can this Image Magick example be translated to Magick.Net?

  magick lena.png -fft  +depth \
          \( -clone 0 -write lena_magnitude.png +delete \) \
          \( -clone 1 -write lena_phase.png +delete \) \
          null:

**Comment by dlemstra** [comment]
There is no support for that due to licensing issues. The fftw library that is used for that is licensed under the GPL license an that is not compatible with the license of this project.
